ramalama/model.py

```diff
@@ -175,7 +175,7 @@ def setup_container(self, args):
         if hasattr(args, "port"):
             conman_args += ["-p", f"{args.port}:{args.port}"]
 
-        if sys.platform == "darwin" and os.path.exists("/dev/dri"):
```

Comment:
@volker48 could you please use: 
```
if not sys.platform == "darwin"
```

Why? It make sure nobody in the future adds it back trying to solve something. BTW, how to reproduce this one?

Reply:
Sure I'll update. To reproduce I just tried to run any image on an M1 Pro Mac using Docker.